assembly/UInt/UInt128.ts

```diff
@@ -19,10 +19,10 @@ import {Codec} from "../interfaces/Codec";
 /** Representation for a UInt128 value in the system. */
 export class UInt128 implements Codec {
```

Comment:
It seems that `UInt128` does not support `fromU8a` `index` definition (reading from a certain byte from the array and not the start).
I think that it would be better if we have consistency across the types so that every type supports the same functionality